*** a/controller/initialize_main_tables.py	2025-10-03 15:07:10.000000000 -0700
--- b/controller/initialize_main_tables.py	2025-10-03 13:48:45.046780456 -0700
***************
*** 1,5 ****
--- 1,8 ----
  import psycopg2
  from psycopg2 import sql
+ import os, sys
+ project_root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+ sys.path.append(project_root)
  from services.config import get_db_connection
  
  def initialize_database_deduplication(enable_index=False):
